app/assets/stylesheets/partials/global.sass

```diff
@@ -179,4 +179,8 @@ ul.share-popover
 
 .moderator
   font-variant: small-caps
-  margin-left: 10px
\ No newline at end of file
+  margin-left: 10px
+
+p
+  img
+    max-width: 100%
```

Comment:
This constrains the redactor upload images to be maximum of 100% – would be good if we could somehow add a class on redactor uploaded images so we don't have to constrain EVERY `img` tag within a `p` tag to max-width: 100% – but this is probably safe for now.